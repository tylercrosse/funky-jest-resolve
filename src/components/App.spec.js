import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

describe('<App />', () => {
  it('should render correctly', () => {
    const renderedComponent = renderer.create(<App />);
    const tree = renderedComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
  
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
  });
});