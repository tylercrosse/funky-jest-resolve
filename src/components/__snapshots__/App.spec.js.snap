exports[`<App /> should render correctly 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg" />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="App-intro">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
  <h1>
    Foo
  </h1>
  <h1>
    Bar
  </h1>
</div>
`;
